firmware/lib/Gantry/Kinematics.h

```diff
@@ -4,12 +4,12 @@
 #include <stdint.h>
 
 typedef struct {
-    uint32_t counts_accel;
-    uint32_t counts_hold;
-    uint32_t counts_decel;
+    uint32_t dist_accel;
```

Comment:
Similar comment - can we clarify the unit of `dist`

Reply:
Ahh, it's described in the comments on `generate_vel_profile` in `Kinematics.cpp`. The units are actually intentionally not specified because it's up to the user. I made it so this `Kinematics` module deals in pure math where it'll use whatever units are used for parameters that get passed to functions.
I can add some comments here to clarify that.